### PR TITLE
CannonKeys DB60 (Daughterboard60) PCB

### PR DESCRIPTION
**Pull request by awkannan**
## Description
New 60% PCB with per key LED spots and underglow.

TODO:
- [x] Test on hardware
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/cannonkeys/db60/rules.mk [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#MCU Name
+MCU = STM32F072
+
+# Build Options
+#   comment out to disable the options.
+#
+
+# project specific files
```
Comment (fauxpark) [inline keyboards/cannonkeys/db60/rules.mk]:
```suggestion
```

File: keyboards/cannonkeys/db60/rules.mk [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#MCU Name
+MCU = STM32F072
+
+# Build Options
+#   comment out to disable the options.
+#
+
```
Comment (fauxpark) [inline keyboards/cannonkeys/db60/rules.mk]:
```suggestion
```

File: keyboards/cannonkeys/db60/rules.mk [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+# project specific files
+
```
Comment (fauxpark) [inline keyboards/cannonkeys/db60/rules.mk]:
```suggestion
```

File: keyboards/cannonkeys/db60/readme.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# CannonKeys DB60
+
+DB60 "Daughterboard 60" Keyboard
+
+Keyboard Maintainer: [Andrew Kannan](https://github.com/awkannan)
```
Comment (fauxpark) [inline keyboards/cannonkeys/db60/readme.md]:
```suggestion
* Keyboard Maintainer: [Andrew Kannan](https://github.com/awkannan)
```

File: keyboards/cannonkeys/db60/readme.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# CannonKeys DB60
+
+DB60 "Daughterboard 60" Keyboard
+
+Keyboard Maintainer: [Andrew Kannan](https://github.com/awkannan)  
+Hardware Supported: STM32F072CBT6
```
Comment (fauxpark) [inline keyboards/cannonkeys/db60/readme.md]:
```suggestion
* Hardware Supported: STM32F072CBT6
```

File: keyboards/cannonkeys/db60/rules.mk [hunk 6]
```diff
@@ -0,0 +1,22 @@
+#MCU Name
```
Comment (fauxpark) [inline keyboards/cannonkeys/db60/rules.mk]:
```suggestion
# MCU Name
```

File: keyboards/cannonkeys/db60/bootloader_defs.h [hunk 7]
```diff
@@ -0,0 +1,7 @@
+/* Address for jumping to bootloader on STM32 chips. */
```
Comment (zvecr) [inline keyboards/cannonkeys/db60/bootloader_defs.h]: This file can be deleted as there is a default at `drivers/boards/GENERIC_STM32_F072XB/bootloader_defs.h` which will be used.

**Comment by awkannan** [comment]
@zvecr @fauxpark Addressed PR Comments

File: keyboards/cannonkeys/db60/rules.mk [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+# Build Options
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE = lite	# Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes	# Mouse keys
+EXTRAKEY_ENABLE = yes	# Audio control and System control
+CONSOLE_ENABLE = yes	# Console for debug
+COMMAND_ENABLE = yes    # Commands for debug and configuration
+SLEEP_LED_ENABLE = yes  # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes	    # USB Nkey Rollover
```
Comment (fauxpark) [inline keyboards/cannonkeys/db60/rules.mk]:
```suggestion
BOOTMAGIC_ENABLE = lite # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes   # Mouse keys
EXTRAKEY_ENABLE = yes   # Audio control and System control
CONSOLE_ENABLE = yes    # Console for debug
COMMAND_ENABLE = yes    # Commands for debug and configuration
SLEEP_LED_ENABLE = yes  # Breathing sleep LED during USB suspend
NKRO_ENABLE = yes       # USB Nkey Rollover
```

File: keyboards/cannonkeys/db60/rules.mk [hunk 9]
```diff
@@ -0,0 +1,22 @@
+# MCU Name
+MCU = STM32F072xB
```
Comment (noroadsleft) [inline keyboards/cannonkeys/db60/rules.mk]:
```suggestion
MCU = STM32F072
```

File: keyboards/cannonkeys/db60/rules.mk [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE = lite	# Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes	# Mouse keys
+EXTRAKEY_ENABLE = yes	# Audio control and System control
+CONSOLE_ENABLE = yes	# Console for debug
+COMMAND_ENABLE = yes    # Commands for debug and configuration
+SLEEP_LED_ENABLE = yes  # Breathing sleep LED during USB suspend
+NKRO_ENABLE = yes	    # USB Nkey Rollover
+CUSTOM_MATRIX = no # Custom matrix file
```
Comment (noroadsleft) [inline keyboards/cannonkeys/db60/rules.mk]:
```suggestion
```

File: keyboards/cannonkeys/db60/info.json [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
...Lock", "x":0, "y":2, "w":1.75}, {"label":"A", "x":1.75, "y":2}, {"label":"S", "x":2.75, "y":2}, {"label":"D", "x":3.75, "y":2}, {"label":"F", "x":4.75, "y":2}, {"label":"G", "x":5.75, "y":2}, {"label":"H", "x":6.75, "y":2}, {"label":"J", "x":7.75, "y":2}, {"label":"K", "x":8.75, "y":2}, {"label":"L", "x":9.75, "y":2}, {"label":":", "x":10.75, "y":2}, {"label":"@", "x":11.75, "y":2}, {"label":"~", "x":12.75, "y":2}, {"label":"Shift", "x":0, "y":3, "w":1.25}, {"label":"|", "x":1.25, "y":3}, {"label":"Z", "x":2.25, "y":3}, {"label":"X", "x":3.25, "y":3}, {"label":"C", "x":4.25, "y":3}, {"label":"V", "x":5.25, "y":3}, {"label":"B", "x":6.25, "y":3}, {"label":"N", "x":7.25, "y":3}, {"label":"M", "x":8.25, "y":3}, {"label":"<", "x":9.25, "y":3}, {"label":">", "x":10.25, "y":3}, {"label":"?", "x":11.25, "y":3}, {"label":"Shift", "x":12.25, "y":3, "w":2.75}, {"label":"Ctrl", "x":0, "y":4, "w":1.25}, {"label":"Win", "x":1.25, "y":4, "w":1.25}, {"label":"Alt", "x":2.5, "y":4, "w":1.25}, {"x":3.75, "y":4, "w":6.25}, {"label":"AltGr", "x":10, "y":4, "w":1.25}, {"label":"Win", "x":11.25, "y":4, "w":1.25}, {"label":"Menu", "x":12.5, "y":4, "w":1.25}, {"label":"Ctrl", "x":13.75, "y":4, "w":1.25}]
```
Comment (noroadsleft) [inline keyboards/cannonkeys/db60/info.json]:
```suggestion
            "layout": [{"label":"\u00ac", "x":0, "y":0}, {"label":"!", "x":1, "y":0}, {"label":"\"", "x":2, "y":0}, {"label":"\u00a3", "x":3, "y":0}, {"label":"$", "x":4, "y":0}, {"label":"%", "x":5, "y":0}, {"label":"^", "x":6, "y":0}, {"label":"&", "x":7, "y":0}, {"label":"*", "x":8, "y":0}, {"label":"(", "x":9, "y":0}, {"label":")", "x":10, "y":0}, {"label":"_", "x":11, "y":0}, {"label":"+", "x":12, "y":0}, {"label":"Backspace", "x":13, "y":0, "w":2}, {"label":"Tab", "x":0, "y":1, "w":1.5}, {"label":"Q", "x":1.5, "y":1}, {"label":"W", "x":2.5, "y":1}, {"label":"E", "x":3.5, "y":1}, {"label":"R", "x":4.5, "y":1}, {"label":"T", "x":5.5, "y":1}, {"label":"Y", "x":6.5, "y":1}, {"label":"U", "x":7.5, "y":1}, {"label":"I", "x":8.5, "y":1}, {"label":"O", "x":9.5, "y":1}, {"label":"P", "x":10.5, "y":1}, {"label":"{", "x":11.5, "y":1}, {"label":"}", "x":12.5, "y":1}, {"label":"Caps Lock", "x":0, "y":2, "w":1.75}, {"label":"A", "x":1.75, "y":2}, {"label":"S", "x":2.75, "y":2}, {"label":"D", "x":3.75, "y":2}, {"label":"F", "x":4.75, "y":2}, {"label":"G", "x":5.75, "y":2}, {"label":"H", "x":6.75, "y":2}, {"label":"J", "x":7.75, "y":2}, {"label":"K", "x":8.75, "y":2}, {"label":"L", "x":9.75, "y":2}, {"label":":", "x":10.75, "y":2}, {"label":"@", "x":11.75, "y":2}, {"label":"~", "x":12.75, "y":2}, {"label":"Enter", "x":13.75, "y":1, "w":1.25, "h":2}, {"label":"Shift", "x":0, "y":3, "w":1.25}, {"label":"|", "x":1.25, "y":3}, {"label":"Z", "x":2.25, "y":3}, {"label":"X", "x":3.25, "y":3}, {"label":"C", "x":4.25, "y":3}, {"label":"V", "x":5.25, "y":3}, {"label":"B", "x":6.25, "y":3}, {"label":"N", "x":7.25, "y":3}, {"label":"M", "x":8.25, "y":3}, {"label":"<", "x":9.25, "y":3}, {"label":">", "x":10.25, "y":3}, {"label":"?", "x":11.25, "y":3}, {"label":"Shift", "x":12.25, "y":3, "w":2.75}, {"label":"Ctrl", "x":0, "y":4, "w":1.25}, {"label":"Win", "x":1.25, "y":4, "w":1.25}, {"label":"Alt", "x":2.5, "y":4, "w":1.25}, {"x":3.75, "y":4, "w":6.25}, {"label":"AltGr", "x":10, "y":4, "w":1.25}, {"label":"Win", "x":11.25, "y":4, "w":1.25}, {"label":"Menu", "x":12.5, "y":4, "w":1.25}, {"label":"Ctrl", "x":13.75, "y":4, "w":1.25}]
```

File: keyboards/cannonkeys/db60/info.json [hunk 12]
```diff
@@ -0,0 +1,19 @@
+{
+    "keyboard_name": "DB60",
+    "url": "https://cannonkeys.com",
+    "maintainer": "awkannan",
+    "width": 12,
```
Comment (noroadsleft) [inline keyboards/cannonkeys/db60/info.json]:
```suggestion
    "width": 15,
```

**Comment by awkannan** [comment]
@zvecr @noroadsleft @fauxpark this one is also ready for another look

**Comment by awkannan** [comment]
@zvecr Would love a quick review!

File: keyboards/cannonkeys/db60/rules.mk [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+
+# Build Options
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE = lite # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes   # Mouse keys
+EXTRAKEY_ENABLE = yes   # Audio control and System control
+CONSOLE_ENABLE = yes    # Console for debug
+COMMAND_ENABLE = yes    # Commands for debug and configuration
+SLEEP_LED_ENABLE = yes  # Breathing sleep LED during USB suspend
```
Comment (zvecr) [inline keyboards/cannonkeys/db60/rules.mk:12]:
```suggestion
SLEEP_LED_ENABLE = no  # Breathing sleep LED during USB suspend
```

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**